tests/tests_pytorch/helpers/datamodules.py

```diff
@@ -20,7 +20,7 @@
 from pytorch_lightning.core.datamodule import LightningDataModule
 from tests_pytorch.helpers.datasets import MNIST, SklearnDataset, TrialMNIST
 
-_SKLEARN_AVAILABLE = RequirementCache("sklearn")
+_SKLEARN_AVAILABLE = RequirementCache("scikit-learn")
```

Comment:
GPU tests timed out. We should check what's the impact of running these tests on CI

Reply:
checking!